taskcluster/ci/visual-metrics/kind.yml

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: fenix_taskgraph.loader.single_dep:loader
+
+kind-dependencies:
+    - browsertime
+    - toolchain
+
+only-for-attributes:
+    - run-visual-metrics
```

Comment:
It's very easy to miss (on line 68 of browsertime's kind.yml): https://github.com/mozilla-mobile/fenix/pull/9087/files#diff-4b34843268cf5d69a8948a547511a40aR68

It's used to determine which tasks need to have a visual-metrics task created for them. We create a separate task to produce visual-metrics because they can be batched there. Otherwise, if we do this processing in the browsertime tasks, their run time will grow quite a bit.

Reply:
Oh, I know why I missed it! I was looking at attributes in `kind.yml`s, and it effectively becomes one at https://github.com/mozilla-mobile/fenix/pull/9087/files#diff-f625d3def67458479916c97dd6c2d1a1R97. I wonder if `run-visual-metrics` should become an actual attribute directly in `kind.yml`. In any case, it's not a big deal and it's not blocking this PR. 

Thanks for the pointer and the context!

Reply:
If you want I can make the change, but I'd prefer to keep these settings as close as possible to what we have in mozilla-central.

Reply:
Let's keep it this way. I prefer to keep the code as close as possible to mozilla-central too.